src/main/java/ay2122s1_cs2103t_w16_2/btbb/logic/parser/util/ParserUtil.java

```diff
@@ -138,6 +156,22 @@ public static GenericString parseGenericString(String genericString, String attr
         return List.of(trimmedKeywords.split("\\s+"));
     }
 
+    /**
+     * Parses a {code String keywords} into a {@code List<Quantity>}.
+     *
+     * @param keywords Keywords to be parsed into Quantity.
+     * @return List of strings containing all the keywords.
+     * @throws ParseException if the given {@code keywords} is invalid.
+     */
+    public static List<Quantity> parseQuantityKeywords(String keywords)
+            throws ParseException {
+        ArrayList<Quantity> quantityKeywords = new ArrayList<>();
+        for (String stringKeyword : parseKeywords(keywords)) {
+            quantityKeywords.add(parseInternalQuantity(stringKeyword));
+        }
+        return quantityKeywords;
+    }
```

Comment:
Keywords is not the right term here also. The method should be like parseQuantities(String input).